package.json

```diff
@@ -58,7 +58,7 @@
     "ganache-cli": "6.1.0",
     "solidity-coverage": "^0.5.0",
     "solium": "^1.1.7",
-    "truffle": "^4.1.8",
+    "truffle": "^4.1.9",
```

Comment:
This should be 4.1.11:
https://github.com/trufflesuite/truffle/releases/tag/v4.1.11